### ISSUE TITLE
Tile: accordion component missing roles, states, properties (4.1.2)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#954]
### Expected behavior
Tiles are accordions which have programmatic roles, states, and properties that can be read by assistive technology. 

See authoring guidance - https://www.w3.org/TR/wai-aria-practices-1.1/#accordion
Examples 
https://w3c.github.io/aria-practices/examples/accordion/accordion1.html
http://heydonworks.com/practical_aria_examples/#progressive-collapsibles

### Actual behavior
Tiles missing this information.
